README.md

```diff
@@ -4,13 +4,91 @@ The [Penumbra](https://penumbra.zone/) monorepo for all things web.
+
+### [`@penumbra-zone/wasm` nextjs example](https://github.com/penumbra-zone/nextjs-penumbra-wasm-example)
+
+## What's in here
+
+### [Minifront](https://app.testnet.penumbra.zone/): Dapp to swap, stake, and send on the Penumbra testnet.
+
+### [Prax](https://chrome.google.com/webstore/detail/penumbra-wallet/lkpmkhpnhknhmibgnmmhdhgdilepfghe): Extension for Chrome that provides key custody, manages chain activity, and hosts services used by dapps.
+
+### [Status](https://grpc.testnet.penumbra.zone/): Public info dashboard for Penumbra nodes.
+
+### Packages
+
+**🌘
+[@penumbra-zone/bech32m](https://www.npmjs.com/package/@penumbra-zone/bech32m) 🌑
+[@penumbra-zone/client](https://www.npmjs.com/package/@penumbra-zone/client) 🌑
+[@penumbra-zone/constants](https://www.npmjs.com/package/@penumbra-zone/constants) 🌑
+[@penumbra-zone/crypto](https://www.npmjs.com/package/@penumbra-zone/crypto) 🌑
+[@penumbra-zone/getters](https://www.npmjs.com/package/@penumbra-zone/getters) 🌑
+[@penumbra-zone/keys](https://www.npmjs.com/package/@penumbra-zone/keys) 🌑
+[@penumbra-zone/perspective](https://www.npmjs.com/package/@penumbra-zone/perspective) 🌑
+[@penumbra-zone/protobuf](https://www.npmjs.com/package/@penumbra-zone/protobuf) 🌑
+[@penumbra-zone/services](https://www.npmjs.com/package/@penumbra-zone/services) 🌑
+[@penumbra-zone/services-context](https://www.npmjs.com/package/@penumbra-zone/services-context) 🌑
+[@penumbra-zone/storage](https://www.npmjs.com/package/@penumbra-zone/storage) 🌑
+[@penumbra-zone/transport-chrome](https://www.npmjs.com/package/@penumbra-zone/transport-chrome) 🌑
+[@penumbra-zone/transport-dom](https://www.npmjs.com/package/@penumbra-zone/transport-dom) 🌑
+[@penumbra-zone/types](https://www.npmjs.com/package/@penumbra-zone/types) 🌑
+[@penumbra-zone/wasm](https://www.npmjs.com/package/@penumbra-zone/wasm)
+🌒**
+
+## Documentation
+
+General documentation is available in [docs/README.md](docs/README.md). Package-specific documentation is available in each respective package.
+
+## Getting Started
+
+To start developing the software in this repository, you simply:
+
+- [Install Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) (probably with rustup)
+- [Install Node.js](https://nodejs.org/en/download/package-manager) (at least version 20)
+- [Install pnpm](https://pnpm.io/installation) (probably via corepack)
+- Install Google Chrome
+
+Once you have all these tools, you can
+
+```sh
+git clone https://github.com/penumbra-zone/web
+cd web
+pnpm i
+pnpm dev
+```
+
+You now have a local copy of Minifront available at
+[`https://localhost:5173`](https://localhost:5173) and an unbundled Prax is
+available at `apps/extension/dist/`, ready to be loaded into your testing
+browser.
+
+Minifront will hot-reload. If you're working on the web extension, Chrome
+supports hot-reloading extension pages, but cannot hot-reload the extension
```

Comment:
> Chrome supports hot-reloading extension pages

Wait, it does? It never hot-reloads for me!

Reply:
it does technically, but it's turned off in webpack, and i've just been refreshing to see the update. i should update the language there